### ISSUE TITLE
A11y focus test fails

### ISSUE THREAD

**Issue by billy-hardy** [PolymerElements/paper-listbox#26]
### Description
a11y focus test fails in suite. (is fine outside of suite)

### Expected outcome
element should be visible if focusable

### Actual outcome
I think due to it being loaded by a suite, the element is not visible

### Live Demo
see travisci

### Steps to reproduce
run tests

### Browsers Affected

- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10
